staging/src/k8s.io/legacy-cloud-providers/vsphere/vclib/utils_test.go

```diff
@@ -70,13 +70,43 @@ func TestUtils(t *testing.T) {
 	}
 }
 
+func TestIsvCenter80NotDeprecated(t *testing.T) {
```

Comment:
Please rework it into table driven test - all the test functions look the same.

Reply:
done.
combined all tests to one test using

```golang
type testsData struct {
	vcVersion    string
	vcAPIVersion string
	isDeprecated bool
}

testdataArray := []testsData{
		testsData{"8.0.0", "8.0.0.0", false},
		testsData{"7.0.3", "7.0.3.0", false},
		testsData{"7.0.2", "7.0.2.0", false},
		testsData{"7.0.1", "7.0.1.1", true},
		testsData{"7.0.0", "7.0.0.0", true},
		testsData{"6.7.0", "6.7.3", true},
		testsData{"6.7.0", "6.7", true},
		testsData{"6.7.0", "6.7.2", true},
		testsData{"6.7.0", "6.7.1", true},
		testsData{"6.5.0", "6.5", true},
	}
```